stable/kommander/values.yaml

```diff
@@ -170,6 +170,16 @@ grafana:
     #   hosts:
     #   - grafana.example.com
 
+  folders: 
+    enabled: true
```

Comment:
Added this in case we need to feature flag this for some reason, but I think adding the namespace is pretty harmless for now